src/commands/forcescore.ts

```diff
@@ -29,7 +28,21 @@ const command: CommandDefinition = {
 		if (scores.length < 2) {
 			throw new UserError("Must provide score in format `#-#` e.g. `2-1`.");
 		}
-		const response = await support.tournamentManager.submitScoreForce(id, player, scores[0], scores[1]);
+		// Check command syntax first to avoid a database round trip
+		const tournament = await support.database.authenticateHost(id, msg.author.id, TournamentStatus.IPR);
+		try {
+			// eslint-disable-next-line no-var
+			var { challongeId } = await support.database.getConfirmedPlayer(player, id);
```

Comment:
is var the only way to establish a variable inside a try block and use it after the catch? that's really frustrating. what about `try { let } finally {}`?

Reply:
`let` and `const` are block-scoped and introduced somewhere around ES5/2015. `var` is the classic JavaScript declaration and it's scoped to the function and hoisted to the top-level and beginning. This was the most natural way I could find for doing this as the assignment I want to perform also involves a destructure.